### ISSUE TITLE
Remote-FTP tree hangs up after removing a folder

### ISSUE THREAD

**Issue by aaaatpdev** [icetee/remote-ftp#1013]
Hi. I have the folder with name "folder" (for example). This folder has about 1800 folders. Each of these folders have 5-6 files inside. After removing the folder with name "folder" (via Remote-FTP tree) Remote-FTP tree hangs up (can't list unsynchronized folder, can't open a file) and in the status bar i have infinitely "Deleting folder". What can cause the issue? In server console ```rm -rf /path/to/folder``` works as expected

Atom 1.20.1
Remote-FTP 1.1.3

**Comment by icetee** [comment]
I think try to calculate files and time left, it is likely that there are too many. Progress bar just spins?

**Comment by aaaatpdev** [comment]
Yeah, it spins infinitely and can't list any folder or open any file. For example, a folder with about 30 directories and 1-2 files inside each of this removes successfully. Listing the folder with about 1800 folders inside is also successfull
